sandbox/public/index.html

```diff
@@ -37,34 +37,34 @@
 
     <script>
       alloy("configure", {
-        orgId: 23784612874,
-        propertyId: 9999999,
+        orgID: 23784612874,
+        propertyID: 9999999,
```

Comment:
In a slack discussion, we discussed aligning how we name ID variables and, since `ID` is used in the platform instead of `Id`, we decided to go with that.